### PR TITLE
[Major] Add "getLastCommand" method to simlplify API

### PR DESCRIPTION
**Pull request by flforget**
_ add "getLastCommand" method to simlplify API
_ "solveTrajectory" now return last solved command, also to simplify API
_ Change convergence criterion for a correct one (convergence when cost stop changing)
_ Add 2 more examples "Linear" and "Inverse pendulum" for future unitTests to come

### CODE REVIEW

File: examples/linear/mainLinear.cpp [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    {
+      u << uList[i];
+      x = model.computeNextState(dt, x, u);
+      x = xList[i];
+      fichier1 << x(0, 0) << "," << x(1, 0) << "," << u(0, 0) << endl;
+    }
+    fichier1.close();
+  }
+  else
+    cerr << "erreur ouverte fichier" << endl;
```
Comment (olivier-stasse) [inline examples/linear/mainLinear.cpp:85]: Thanks ! All the code is in English but the error messages are in French.
Could you fix that ?

**Review by olivier-stasse — APPROVED** [review]
I am fine with the pull request

File: examples/linear/mainLinear.cpp (hunk 1 above)
Comment (flforget) [reply examples/linear/mainLinear.cpp:85]: I'm preparing another PR for code refactoring, I will do the translations at the same time

Comment (olivier-stasse) [reply examples/linear/mainLinear.cpp:85]: thx.

**Pull request merged**